Cargo.toml

```diff
@@ -38,6 +38,48 @@ default-members = [
     "libparsec/crates/types",
 ]
 
+[workspace.dependencies]
+anyhow = { version = "1.0.71" }
```

Comment:
we need `default-feature = false` here, otherwise using `default-feature = false` in a sub `cargo.toml` would be silently ignored :/